### PR TITLE
Added signal and freeheap stats and set default value for property name

### PR DESCRIPTION
**Pull request by elbowz**
Little adds and fixes as explained in the PR title

About the `free-heap` stats, I have only fixed it, but I don't know how you had already in your code, because it was added a couple of days ago ([PR 719](https://github.com/homieiot/homie-esp8266/pull/719))...maybe a re-add in the homie!?

Default property name, it's quite useful if optional topic `$name` is not set (as in my case).

Thanks for this great project!

p.s. feel free to reject the PR for any reason

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -339,8 +340,10 @@ function onMessageArrived(message) {
         devices.deviceList[device_id]["stats_uptime"] = payload;
       } else if (topic[2] === "interval") {
         devices.deviceList[device_id]["stats_interval"] = payload;
-      } else if (topic[2] === "freeheap") {
+      } else if (topic[2] === "free-heap") {
```
Comment (qwandor) [inline js/app.js:343]: According to the [specification](https://github.com/homieiot/convention/blob/develop/extensions/documents/homie_legacy_stats_extension.md) it should be `freeheap` not `free-heap`.

Comment (elbowz) [reply js/app.js:343]: You have right, I didn't see the specification. But that is related to Homie 4.0 or also to 3.0.1?

As you can see here: https://github.com/elbowz/homie-esp8266/blob/c4ed8ec33e3c5f3ce5f651a2b903be28e865dacf/src/Homie/Boot/BootNormal.cpp#L198 now Homie-esp8266 works with `free-heap`.

But I guess the right way is fix on Homie-esp8266

**Comment by luebbe** [comment]
Hmm, I accepted PR #719 a while ago, but didn't check the specs. If they say freeheap instead of free-heap, we should fix it.

File: js/app.js (hunk 1 above)
Comment (qwandor) [reply js/app.js:343]: In 4.0 stats are an extension rather than part of the core spec. The stats extension specification is what I linked above.

**Pull request closed without merging**